contracts/5/uniswap/UniswapV2Library.sol

```diff
@@ -18,20 +19,21 @@ library UniswapV2Library {
     address factory,
     address tokenA,
     address tokenB
-  ) internal pure returns (address pair) {
-    (address token0, address token1) = sortTokens(tokenA, tokenB);
-    pair = address(
-      uint256(
-        keccak256(
-          abi.encodePacked(
-            hex'ff',
-            factory,
-            keccak256(abi.encodePacked(token0, token1)),
-            hex'b594b680ac8ffbcd817033d8b9a0f18807d93811006cb8a3ad1884015bb00ea0' // init code hash
-          )
-        )
-      )
-    );
+  ) internal view returns (address pair) {
+    return IUniswapV2Factory(factory).getPair(tokenA, tokenB); // For easy testing
```

Comment:
Do we intend to change this back?

Reply:
Nope. Hardhat has some issue with the fix creationCode, so I need to change everytime I recompile.